src/main/kotlin/no/nav/syfo/service/JournalpostService.kt

```diff
@@ -26,6 +28,9 @@ class JournalpostService(
     private fun hentInngaendeJournalpost(gsakId: String, inntektsmelding: Inntektsmelding): InngaendeJournalpost {
         val inngaaendeJournal = inngaaendeJournalConsumer.hentDokumentId(inntektsmelding.journalpostId)
         val behandlendeEnhet = behandlendeEnhetConsumer.hentBehandlendeEnhet(inntektsmelding.fnr, inntektsmelding.id)
+        val arbeidsgiverNavn = inntektsmelding.arbeidsgiverOrgnummer?.let {
+            brregClient.getVirksomhetsNavn(inntektsmelding.arbeidsgiverOrgnummer)
+        } ?: "Ukjent "
```

Comment:
Burde stå Arbeidsgiver just in case

Reply:
Rettelse. Den burde være null siden du defaulter til "Arbeidsgiver" i InngaendeJournalpost klassen